src/ResourceManager/Reservations/Commands.Reservations/Cmdlets/SplitReservation.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.Reservations.Common;
+using Microsoft.Azure.Management.Reservations.Models;
+using System.Management.Automation;
+using Microsoft.Azure.Management.Reservations;
+using Microsoft.Azure.Commands.Reservations.Models;
+
+namespace Microsoft.Azure.Commands.Reservations.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Split, "AzureRmReservation", DefaultParameterSetName = Constants.ParameterSetNames.CommandParameterSet, SupportsShouldProcess = true), OutputType(typeof(List<PSReservation>))]
+    public class SplitReservation : AzureReservationsCmdletBase
+    {
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.CommandParameterSet,
+            Mandatory = true)]
+        [ValidateNotNull]
+        public string ReservationOrderId { get; set; }
+
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.CommandParameterSet,
+            Mandatory = true)]
+        [ValidateNotNull]
+        public string ReservationId { get; set; }
+
+        [Parameter(Mandatory = true)]
+        [ValidateNotNull]
+        [ValidateCount (2, 2)]
+        public int?[] Quantity { get; set; }
+
+        [Parameter(ParameterSetName = Constants.ParameterSetNames.ObjectParameterSet,
+            Mandatory = true,
+            ValueFromPipeline = true)]
+        [ValidateNotNull]
+        public PSReservation Reservation { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            if (ShouldProcess("AzureRmReservation", "Split"))
```

Comment:
Again, what would you want to tell the user about this action, if they are runnign this cmdlet with -WhatIf.  What should the user know about the effects of this operation?

Reply:
addressed in above comment.

Reply:
Same comment as above.